src/subdomains/core/buy-crypto/process/services/buy-crypto.service.ts

```diff
@@ -344,6 +350,20 @@ export class BuyCryptoService {
     return entity;
   }
 
+  async refundBuyCrypto(buyCryptoId: number): Promise<void> {
+    const buyCrypto = await this.buyCryptoRepo.findOne({ where: { id: buyCryptoId }, relations: { checkoutTx: true } });
+    if (!buyCrypto.checkoutTx) throw new BadRequestException('Return is only supported with checkoutTx');
+
+    const chargebackRemittanceInfo = await this.checkoutService.refundPayment(buyCrypto.checkoutTx.paymentId);
+
+    buyCrypto.chargebackDate = new Date();
+    buyCrypto.chargebackRemittanceInfo = JSON.stringify(chargebackRemittanceInfo);
```

Comment:
Can we just use the reference for `chargebackRemittanceInfo` instead of storing the whole thing as JSON?